sed/config/flash_example_config.yaml

```diff
@@ -139,6 +182,8 @@ dataframe:
   # (Not to be changed by user)
   beamtime_dir:
     pg2: "/asap3/flash/gpfs/pg2/"
+    hextof: "/asap3/fs-flash-o/gpfs/hextof/"
```

Comment:
again, adding this here right now doesnt make sense I think

Reply:
@zain-sohail can you please set this here and in the notebook to something useful and resolve? I don't have any idea about the fs structure on maxwell, nor where I could have write access.